tests/storage_adapter_tests/test_sqlalchemy_adapter.py

```diff
@@ -347,10 +338,9 @@ def test_response_list_in_results(self):
 
 
 class ReadOnlySQLAlchemyDatabaseAdapterTestCase(SQLAlchemyAdapterTestCase):
-
     def setUp(self):
         """
-        Make the adapter writable before every test.
+        Make the adapter read only before every test.
```

Comment:
I don't believe this change to the comment is correct. It would be making it read only if it was setting `read_only = True`, however the line below is `self.adapter.read_only = False`.